doc/changes/changes_3.4.1.md

```diff
@@ -6,10 +6,27 @@ Code name: Workaround for log rotation bug in docker-db
 
 In the 3.4.1 release we added a mitigation for a bug in [`docker-db`](https://github.com/exasol/docker-db) that causes log rotation to break after 40 minutes. Since ETC depends on reading the logs, this mitigation is necessary for running the container longer than 40 minutes.
 
+Since this change required longer running integration tests, we added three build profiles to the Maven configuration:
+
+* fast
+* slow
+* expensive
+
+"Fast" means all test that don't take longer than a second or two. "Slow" test can run for a handful of minutes and "expensive" tests run longer than 10 minutes.
+
+The default is now "fast" for regular builds and "slow" for Travis CI builds.
```

Comment:
What is a regular build?

Reply:
Any build you start that is not a CI build.